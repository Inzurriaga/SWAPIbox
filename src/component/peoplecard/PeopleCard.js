import React, { Component } from "react"

export default class PeopleCard extends Component{
    constructor(){
        super();
        this.state = {
            test: true
        } 
    }

    render(){
        return(
            <div className="people-card">
               <h3>{this.props.people.name}</h3>
               <p>species: {this.props.people.speciesName}</p>
               <p>world: {this.props.people.worldName}</p>
               <p>population: {this.props.people.population}</p>
            </div>
        )
    }
}