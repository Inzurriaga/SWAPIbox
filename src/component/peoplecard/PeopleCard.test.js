import React from "react"
import { shallow } from "enzyme"
import PeopleCard from "./PeopleCard"

const mockdata = {
    name: "bob",
    speciesName: "human",
    worldName: "earth",
    population: "300"
}


describe("people_card", () => {
    let wrapper;
    beforeEach(() => {
        wrapper = shallow(
            <PeopleCard people={mockdata}/>
        )
    })
    it("should match snapshot", () => {
        expect(wrapper).toMatchSnapshot()
    })
})