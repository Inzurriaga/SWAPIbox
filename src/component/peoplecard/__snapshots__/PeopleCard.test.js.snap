// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`people_card should match snapshot 1`] = `
<div
  className="people-card"
>
  <h3>
    bob
  </h3>
  <p>
    species: 
    human
  </p>
  <p>
    world: 
    earth
  </p>
  <p>
    population: 
    300
  </p>
</div>
`;
